File: Week 01/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    // printf("Stored: %i", n);
    printf("\n");
    for (int k = 1; k <= n; k++)
    {
        for (int j = n - k - 1; j >= 0; j--)
        {
            printf(" ");
        }
        for (int i = k; i >= 1; i--)
        {
            printf("#");
        }
        printf("  ");
        for (int i = k; i >= 1; i--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 06/mario.py
from cs50 import get_int

while True:
    height = get_int("Height :")
    if height > 0 and height < 9:
        break

for i in range(height):
    # a is number of spaces
    a = height - 1 - i
    # b is number of hashes
    b = i + 1
    print((" "*(a))+("#"*(b))+("  ")+("#"*(b)), end='\n')
